index.bs

```diff
@@ -1416,26 +1417,26 @@ Create a named {{MLOperand}} based on a descriptor, that can be used as an input
 ### constant operands ### {#api-mlgraphbuilder-constant}
 Create a constant {{MLOperand}} that can be used in {{MLGraphBuilder}} methods.
 
-#### {{MLGraphBuilder/constant(descriptor, bufferView)}} #### {#api-mlgraphbuilder-constant-bufferview}
+#### {{MLGraphBuilder/constant(descriptor, bufferSource)}} #### {#api-mlgraphbuilder-constant-buffersource}
 Create a constant {{MLOperand}} of the specified data type and shape that contains the initializing data.
 
-<div dfn-for="MLGraphBuilder/constant(descriptor, bufferView)" dfn-type=argument>
+<div dfn-for="MLGraphBuilder/constant(descriptor, bufferSource)" dfn-type=argument>
     **Arguments:**
         - <dfn>descriptor</dfn>: an {{MLOperandDescriptor}}. The descriptor of the output tensor.
-        - <dfn>bufferView</dfn>: an {{ArrayBufferView}}. The view of the buffer containing the initializing data.
+        - <dfn>bufferSource</dfn>: an {{AllowSharedBufferSource}}. The buffer containing the initializing data.
     **Returns:** an {{MLOperand}}. The constant output tensor.
 </div>
 
 <details open algorithm>
   <summary>
-    The <dfn method for=MLGraphBuilder>constant(|descriptor|, |bufferView|)</dfn> method steps are:
+    The <dfn method for=MLGraphBuilder>constant(|descriptor|, |bufferSource|)</dfn> method steps are:
   </summary>
     1. If [=this=].{{MLGraphBuilder/[[hasBuilt]]}} is true, then [=exception/throw=] an "{{InvalidStateError}}" {{DOMException}}.
     1. If [=MLOperandDescriptor/checking dimensions=] given |descriptor| returns false, then [=exception/throw=] a {{TypeError}}.
-    1. If [=validating buffer with descriptor=] given |bufferView| and |descriptor| returns false, then [=exception/throw=] a {{TypeError}}.
+    1. If [=validating buffer with descriptor=] given |bufferSource| and |descriptor| returns false, then [=exception/throw=] a {{TypeError}}.
```

Comment:
Note: This depends on #787 for supporting `AllowSharedBufferSource` in "validate buffer with descriptor" algorithm.

Reply:
https://github.com/webmachinelearning/webnn/pull/787 is now complete.